code/modules/spells/spell_types/vampire_powers.dm

```diff
@@ -104,24 +109,23 @@
 
 
 /obj/effect/proc_holder/spell/targeted/hypnotise
-	name = "Hypnotize (20)"
-	desc= "A piercing stare that incapacitates your victim for a good length of time."
+	name = "Hypnotize (30)"
+	desc= "A piercing stare that knocks out your victim for a good length of time, and makes them forget what recently happened."
 	action_icon_state = "hypnotize"
-	blood_used = 20
+	blood_used = 30
 	action_icon = 'icons/mob/vampire.dmi'
 	action_background_icon_state = "bg_demon"
 	vamp_req = TRUE
-
+//Todo, make the targeted abilities auto activate if theres only one target in range.
 /obj/effect/proc_holder/spell/targeted/hypnotise/cast(list/targets, mob/user = usr)
 	for(var/mob/living/target in targets)
 		user.visible_message("<span class='warning'>[user]'s eyes flash briefly as he stares into [target]'s eyes</span>")
-		if(do_mob(user, target, 20))
+		if(do_mob(user, target, 40))
 			to_chat(user, "<span class='warning'>Your piercing gaze knocks out [target].</span>")
 			to_chat(target, "<span class='warning'>You find yourself falling asleep.</span>")
 			target.SetSleeping(400) //So its actually usefull for abducting people, should be enough to drag them off and cuff them and remove their headset.
-			//Todo, mute after the sleep and stun the target long enough for the sleep.
-
-
+			sleep(400)
+			to_chat(target, "<span class='warning'>You wake up with a wicked headache and dont remember what happened recently.</span>") //Maybe give them the trauma that gives them a objective about what they hear? For now, thisll hopefully let the vampire suck someone without having to kill them.
```

Comment:
`addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, target, "<span class='warning'>You wake up with a wicked headache and dont remember what happened recently.</span>") 40 SECONDS)`

Don't use sleep().

Also this seems pretty stupid, not the commented idea, just the timed message per se.

Reply:
Hmm, perhaps i would try to use hypnotize here, the one that TG with a flash, and the spell itself is called hypnotize.